### ISSUE TITLE
Missing text equivalent for images (PDF documents)

### ISSUE THREAD

**Issue by dannychapman** [department-of-veterans-affairs/vets-website#861]
https://www.vets.gov/playbook/downloads/Voices_Of_Veterans.pdf
https://www.vets.gov/playbook/downloads/VHA_Style_Guide_508.pdf
